features/register.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Order placement using Logged in customer
+
+
+Scenario: Order Placement using (negative scenario)
+    Given The user will navigates to HomepageURL.
+    When User clicks on the Sigin button
+    Then The user should be navigated to Sigin page
+    When The user clicks on Sigin button
+    Then the user should see 2 error message
```

Comment:
@karthikeyan7010 We can't be generic in testing. You've to specify exact error you're expecting and create multiple test scenarios based on the validation